nullaway/src/main/java/com/uber/nullaway/NullabilityUtil.java

```diff
@@ -271,11 +271,17 @@ public static Stream<? extends AnnotationMirror> getAllAnnotationsForParameter(
                         && t.position.parameter_index == paramInd));
   }
 
-  private static Stream<? extends AnnotationMirror> getTypeUseAnnotations(Symbol symbol) {
+  public static Stream<? extends AnnotationMirror> getTypeUseAnnotations(Symbol symbol) {
```

Comment:
Why the change to public?

Reply:
Oh, for debugging. I just missed reverting this one. Good catch! :)